test-app/app/Global.java

```diff
@@ -1,73 +0,0 @@
-import com.feth.play.module.pa.PlayAuthenticate;
-import com.feth.play.module.pa.PlayAuthenticate.Resolver;
-import com.feth.play.module.pa.exceptions.AccessDeniedException;
-import com.feth.play.module.pa.exceptions.AuthException;
-
-import play.Application;
-import play.GlobalSettings;
-import play.mvc.Call;
-
-import controllers.routes;
-
-public class Global extends GlobalSettings {
-
-	@Override
-	public void onStart(final Application app) {
-		PlayAuthenticate.setResolver(new Resolver() {
-
-			@Override
-			public Call login() {
-				// Your login page
-				return controllers.routes.Application.login();
-			}
-
-			@Override
-			public Call afterAuth() {
-				// The user will be redirected to this page after authentication
-				// if no original URL was saved
-				return controllers.routes.Application.index();
-			}
-
-			@Override
-			public Call afterLogout() {
-				return controllers.routes.Application.index();
-			}
-
-			@Override
-			public Call auth(final String provider) {
-				// You can provide your own authentication implementation,
-				// however the default should be sufficient for most cases
-				return com.feth.play.module.pa.controllers.routes.Authenticate
-						.authenticate(provider);
-			}
-
-			@Override
-			public Call onException(final AuthException e) {
-				if (e instanceof AccessDeniedException) {
-					return routes.Application
-							.oAuthDenied(((AccessDeniedException) e)
-									.getProviderKey());
-				}
-
-				// more custom problem handling here...
-
-				return super.onException(e);
-			}
-
-			@Override
-			public Call askLink() {
-				// We don't support moderated account linking in this sample.
-				// See the play-authenticate-usage project for an example
-				return null;
-			}
-
-			@Override
-			public Call askMerge() {
-				// We don't support moderated account merging in this sample.
-				// See the play-authenticate-usage project for an example
-				return null;
-			}
-		});
-	}
-
-}
```

Comment:
I had it stashed - uploaded this change to PR - it's here: https://github.com/pdolega/play-authenticate/blob/play-2.5/test-app/app/auth/AuthInitializer.java

The PR is still work in progress (I actually stopped squashing commits anymore - it may be easier for you to see actual changes as the work goes; I'll squash them at the end).